testing/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+set -e
+
+echo "Expect to run from docker, in callek/shipit-test-runner"
+echo "  docker run --rm -v `pwd`:/src -ti callek/shipit-test-runner /src/testing/run_tests.sh"
```

Comment:
I'm really confused about the purpose of this script if it's not for running tests.